### DISCUSSION TITLE
Site works locally but not on GitHub Pages

### DISCUSSION THREAD

**Discussion by harpalsahota** [mmistakes/minimal-mistakes#2448] [Q&A]
Most likely a noob question. I've scoured the internet and the issue list but I can't seem to find a solution to my problem. The theme works locally but when uploaded to github I'm hit with a 404 see image below:

![Untitled](https://user-images.githubusercontent.com/9675427/77013036-20240180-6967-11ea-8b36-dd35e1c8c9b2.png)

My github page with the code can be found [here](https://github.com/harpalsahota/harpalsahota.github.io). Any idea what could be the issue?

The url for my page is [here](https://harpalsahota.github.io/)

**Comment by mmistakes** [comment]
Other than you have no posts yet, your site looks fine to me.
You probably didn’t give GitHub enough time to finish building your site. That first build isn’t instant.

https://harpalsahota.github.io/

**Comment by harpalsahota** [comment]
I think I'm getting somewhere but I don't know if what I'm doing is correct. I've moved content out of `_site/home.md` into `index.md` which is the root directory. The page no longer shows a 404 but the content is not loading. Again this works fine locally, but not on pages.

This is what it looks like locally for me:

![Untitled](https://user-images.githubusercontent.com/9675427/77017137-8746b380-6971-11ea-9956-247d571488bf.png)

**Comment by mmistakes** [comment]
Check your content. You have this page under _pages that is likely causing havoc with your index.md file

https://github.com/harpalsahota/harpalsahota.github.io/blob/master/_pages/_.md

They both have the same permalink and are overwriting each other.
